homeassistant/components/roomba/config_flow.py

```diff
@@ -24,7 +24,7 @@
 
 ROOMBA_DISCOVERY_LOCK = "roomba_discovery_lock"
 ALL_ATTEMPTS = 2
-HOST_ATTEMPTS = 6
+HOST_ATTEMPTS = 2
```

Comment:
Changing this will increase the chance the roomba is missed because it is won't be long enough for the device to see if due to its sleep patterns.